tcms/testruns/templates/testruns/get.html

```diff
@@ -30,6 +30,15 @@ <h1 class="col-md-12" style="margin-top: 0; {% if object.stop_date %}text-decora
     <div class="row row-cards-pf">
         <div class="col-xs-12 col-sm-12 col-md-3">
             <div class="card-pf card-pf-accented card-pf-aggregate-status">
+                <h2 class="card-pf-title" style="text-align: left">
+                    <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
+                    <a href="{% url 'testruns-search' %}?product={{ object.plan.product_id }}" title="Planned start {{ object.planned_start }}">{{ object.planned_start}}</a>
+                </h2>
+
+                <h2 class="card-pf-title" style="text-align: left">
+                    <span class="fa fa fa-calendar-o"></span>{% trans 'Planned stop' %}:
+                    <a href="{% url 'testruns-search' %}?product={{ object.plan.product_id }}" title="Planned stop {{ object.planned_stop }}">{{ object.planned_stop}}</a>
```

Comment:
Remove search URL. Moreover as-is it is incorrect anyway b/c it points to the product.